app/assets/javascripts/discourse/app/components/d-modal.hbs

```diff
@@ -21,9 +17,9 @@
     {{will-destroy this.cleanupListeners}}
     {{trap-tab preventScroll=false}}
   >
-    <div class="modal-outer-container">
```

Comment:
So I put middle-container back because what you said last time that it was often used. So it's only outer- and -inner container that's really gone, but they were used less I thought.

Reply:
interstitials plugin uses the new d-modal already, so that shouldn't cause any big issues thankfully.